packages/gallery/src/components/gallery/proGallery/slideshowView.js

```diff
@@ -882,6 +882,9 @@ class SlideshowView extends React.Component {
         getActiveItemIndex: () => {
           return this.state.activeIndex;
         },
+        getActiveGroupIndex: () => {
+          return this.getCenteredItemOrGroupIdxByScroll('groups');
```

Comment:
Why would this be different than line 883?
Why isnt this a state? this calculation should not be triggered by request, but by change.